src/pages/index.tsx

```diff
@@ -1,18 +1,33 @@
+import { useEffect, useState } from 'react'
 import Link from 'next/link'
 
 import Header from '@/components/Header'
 import Summary from '@/components/Home/summary'
 import NavBar from '@/components/NavBar'
+import { withProtected } from '@/components/PrivateRoute'
 import TopNav from '@/components/TopNav'
 import Button from '@/components/UI/button'
 import useUser from '@/hooks/user'
 
 const Home = () => {
-  const { isSuccess, data: currentUser } = useUser()
+  const { data: currentUser } = useUser()
 
-  const welcomeMessage = isSuccess
-    ? `Welcome, ${currentUser?.name}!`
-    : 'Welcome!'
+  const [welcomeMessage, setWelcomeMessage] = useState('Welcome!')
+  const [numVisits, setNumVisits] = useState(0)
+  const [numHours, setNumHours] = useState(0)
+  const [commutedDist, setCommutedDist] = useState(0)
+  const [walkedDist, setWalkedDist] = useState(0)
+
+  useEffect(() => {
```

Comment:
Likewise, there's no need to call `useEffect` because the component will re-render as expected when the values from `useUser` change (i.e. from a refetch or some state mution).